tianshou/utils/optim.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional
+
+import torch
+from torch import nn
+
+
+def optim_step(
+    loss: torch.Tensor,
+    optim: torch.optim.Optimizer,
+    module: nn.Module,
+    max_grad_norm: Optional[float] = None,
+) -> None:
+    """Perform a single optimization step.
+
+    :param loss:
+    :param optim:
+    :param module:
```

Comment:
missing docs?

Reply:
figured the names are self explanatory. If you want, I can add something, but it felt wrong to write 
`:param loss: a loss function` etc.